Packs/Absolute/Integrations/Absolute/Absolute.py

```diff
@@ -1106,7 +1105,12 @@ def get_device_command(args, client) -> CommandResults:
     else:
         query_string = parse_return_fields(",".join(DEVICE_GET_COMMAND_RETURN_FIELDS), query_string)
 
-    res = client.api_request_absolute("GET", "/v3/reporting/devices", query_string=query_string)
+    res = client.api_request_absolute(
+        "GET",
+        "/v3/reporting/devices",
+        query_string=query_string,
+        page_size=500,
```

Comment:
lets put as a constant at the top of the file, with a comment explaining why 500